src/com/ortiz/touch/TouchImageView.java

```diff
@@ -133,22 +133,23 @@ private void sharedConstructing(Context context) {
         setScaleType(ScaleType.MATRIX);
         setState(State.NONE);
         onDrawReady = false;
-        super.setOnTouchListener(new PrivateOnTouchListener());
+        super.setOnTouchListener(new PrivateOnTouchListener());
+        setImageBitmap(null);

```

Comment:
Only change is adding a dummy setImageBitmap call to the initializer.

Reply:
Do you have an example stack trace ?